DasharoPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c

```diff
@@ -1190,6 +1190,11 @@ WarnIfFirmwareUpdateMode (
       );
   ASSERT_EFI_ERROR (Status);
 
+  // Don't bother with warning if FUM was triggered by capsule update

```

Comment:
Nit: this is more in the line of physical user confirmation than a warning.

Reply:
Fixed in https://github.com/Dasharo/edk2/pull/243/commits/94aa71c1af60b56da11662680c9d5e1c92e7162a